src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -40,8 +39,35 @@ class CentipedeError(Exception):
   """Base exception class."""
 
 
+def _find_llvm_symbolizer() -> Optional[str]:
+  """Locate llvm-symbolizer from various sources."""
+  # llvm-symbolizer should be in both BUILD_DIR or EXTRA_BUILD_DIR.
+  build_dir = environment.get_value('BUILD_DIR', fuzzer_utils.EXTRA_BUILD_DIR)
+  llvm_symbolizer_path = os.path.join(build_dir, 'llvm-symbolizer')
+  if os.path.isfile(llvm_symbolizer_path):
+    return llvm_symbolizer_path
+  logs.log_warn('Unable to find llvm-symbolizer at BUILD_DIR or '
+                f'EXTRA_BUILD_DIR: {llvm_symbolizer_path}')
+
+  llvm_symbolizer_path = environment.get_llvm_symbolizer_path()
+  if llvm_symbolizer_path and os.path.isfile(llvm_symbolizer_path):
+    return llvm_symbolizer_path
+  logs.log_warn('Unable to find llvm-symbolizer at: '
+                f'{environment.get_llvm_symbolizer_path()}')
+  return None
+
+
+def _add_llvm_symbolizer():
+  """Add llvm-symbolizer to PATH."""
+  llvm_symbolizer_path = _find_llvm_symbolizer()
+  if llvm_symbolizer_path is not None:
+    # Copy it to /bin to ensure Centipede can use it.
+    shutil.copy(llvm_symbolizer_path, '/bin/')
```

Comment:
This might be a bit hacky, but I think it should work after testing it in OSS-Fuzz cloud VMs.

Reply:
Eh. Why don't you actually add it to the PATH env var?